### PR TITLE
Update oncokb java api client and mapper configs

### PR DESCRIPTION
**Pull request by leexgh**
Fix: https://github.com/genome-nexus/genome-nexus/issues/678

- We need to update oncokb java api client to match their response. 
- Updated oncokb test.

### CODE REVIEW

File: service/src/main/java/org/cbioportal/genome_nexus/service/transformer/ExternalResourceTransformer.java [hunk 1]
```diff
@@ -50,11 +53,8 @@ private List<T> mapJsonToInstance(DBObject rawJson, Class<T> type, ObjectMapper
     {
         List<T> list = new ArrayList<>();
         ObjectMapper mapper = objectMapper;
-
-        if (mapper == null)
-        {
-            mapper = new ObjectMapper();
-        }
+        mapper.enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```
Comment (leexgh) [inline service/src/main/java/org/cbioportal/genome_nexus/service/transformer/ExternalResourceTransformer.java:56]: I need to add this because there is an enum mapping issue:
`"Cannot deserialize value of type `org.oncokb.client.Query$ReferenceGenomeEnum` from String "GRCh37": not one of the values accepted for Enum class: [GRCH37, GRCH38] at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: org.oncokb.client.IndicatorQueryResp["query"]>org.oncokb.client.Query["referenceGenome"])"`

**Comment by leexgh** [comment]
@jeffquinn-msk Yes the problem is in deserializing. I believe the failing test is not related, it failed in other pull requests before.

**Comment by qwwqwwq** [comment]
The last PR made that was merged, which was exclusively a change to markdown documentation files, had two builds run on circleCI, one where this same test case failed and one where it passed: https://app.circleci.com/pipelines/github/genome-nexus/genome-nexus/87

That indicates to me that the test is useless, or there is something very very wrong with the surefire configuration:

Taking a quick look at it:

```
      <plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-surefire-plugin</artifactId>
        <version>3.0.0-M3</version>
        <configuration>
          <forkCount>3</forkCount>
          <reuseForks>true</reuseForks>
          <argLine>-Xmx1024m</argLine>
          <systemPropertyVariables>
            <projectVersion>1.0</projectVersion>
          </systemPropertyVariables>
        </configuration>
      </plugin>
```

it seems a bit suspicious to me, running integration tests in parallel is a notorious source of flakiness, even more so when they involves a real database (as these tests do).

All that being said I have no idea how this particular error could result from a race condition since it seems like the test should never pass given in the input.

Anyway tl;dr I can't approve a PR when the tests are failing, but happy to be overruled if you really aren't concerned. One option to proceed that would get my approval would be to slap the old `@Ignore` on this test case for now.

**Comment by leexgh** [comment]
I have another [PR](https://github.com/genome-nexus/genome-nexus/pull/691) to fix the tests, merge this one
